spark-icons/src/main/res-iconography/arrows/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="design_ic_close_path_data">M23.468,20.9L14.567,12l8.9,-8.901a1.825,1.825 0,0 0,0 -2.548,1.812 1.812,0 0,0 -2.565,0L12,9.434 3.099,0.532A1.815,1.815 0,1 0,0.532 3.099l8.902,8.9 -8.902,8.902a1.815,1.815 0,1 0,2.567 2.567l8.9,-8.901 8.903,8.9c0.71,0.71 1.856,0.71 2.566,0 0.71,-0.71 0.71,-1.856 0,-2.566</string>
```

Comment:
Not sure why we have this string ressource extracted for only a single re-use 🤔

Reply:
Fixed in [2b69e24](https://github.com/adevinta/spark-android/pull/132/commits/2b69e240bf9b2c2cd1b5cb0a129beef17cb06f34)